views/export/preview-ppt2.jade

```diff
@@ -4,6 +4,8 @@ block content
     include ../topnav
     h1 Preview
     #exportOption
+    include exportButton
+    hr(style='height:2px; background-color:#000000;')
```

Comment:
since you're consistently using this style for the hr, would you rather just include it as a class in a css file that is loaded instead of specifying the style individually each time?